Update virtual-network-manage-peering.md

stegag opened the pull request:
As per email discussion with Anavi, You cannot use the "use remote gateways" setting if you already have a gateway configured in your virtual network.

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

stegag committed:
Update virtual-network-manage-peering.md

As per email discussion with Anavi, You cannot use the "use remote gateways" setting if you already have a gateway configured in your virtual network.
